File: A Friend/A Friend/Account.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Drawing;

namespace A_Friend
{
    public class Account
    {
        public string username;
        public string name;
        public string id;
        public byte state;
        public bool priv = false;

        public Account() 
        {

        }
        public Account(string username, string name, string id, byte state) 
        { 
            this.username = username;
            this.name = name;
            this.id = id;
            this.state = state;
        }
    }
}

File: A Friend Java/AFriendJava/src/main/java/com/mycompany/afriendjava/Account.java
package com.mycompany.afriendjava;

public class Account {
    public String username;
    public String name;
    public String id;
    public byte state;
    public byte type;
    public boolean priv = false;

    public Account() 
    {

    }
    public Account(String username, String name, String id, byte state, byte type) 
    { 
        this.username = username;
        this.name = name;
        this.id = id;
        this.state = state;
        this.type = type;
    }
}
